components/brave_wallet_ui/stories/wallet-concept.tsx

```diff
@@ -458,14 +467,17 @@ export const _DesktopWalletConcept = (args: { onboarding: boolean, locked: boole
             toAmount={toAmount}
             fromAssetBalance={fromAssetBalance}
             toAssetBalance={toAssetBalance}
+            toAddress={toAddress}
+            onSetToAddress={onSetToAddress}
             onSetFromAmount={onSetFromAmount}
             onSetToAmount={onSetToAmount}
+            onSubmitSend={onSubmitSend}
             onSubmitSwap={onSubmitSwap}
             flipSwapAssets={flipSwapAssets}
             onSelectNetwork={onSelectNetwork}
             onSelectAccount={onSelectAccount}
             onToggleOrderType={onToggleOrderType}
-            onSelectSwapAsset={onSelectSwapAsset}
+            onSelectAsset={onSelectTransactAsset}
```

Comment:
Do we have a shared state for swap and send here, for the common fields? If so, I wonder if this is a good idea. Fine for now, and we can separate the two if the need arises in future. What's your opinion on this?

Reply:
Added a `sendAmount` state and `onSetSendAmount` method to separate from `fromAmount` state and onSetFromAmount` method.

Reply:
Cool!